documents/knowledge/elements.yaml

```
@@ -1594,9 +1594,10 @@
   description: Join a list by the empty string
   arity: 1
   overloads:
-      fun: First integer for which a(x) is truthy
-      any: Join by empty string
       num: abs(a) <= 1
+      str: pad with 0s to nearest multiple of 8
+      lst: "".join(a)
```

You need to escape the quotation marks here, it's invalid YAML. That's what's making the CIs fail.

```suggestion
      lst: \"\".join(a)
```